openpype/hosts/fusion/api/menu.py

```diff
@@ -145,11 +142,8 @@ def deregister_all_callbacks(self):
     def on_workfile_clicked(self):
         host_tools.show_workfiles()
 
-    def on_create_clicked(self):
-        host_tools.show_creator()
```

Comment:
we could still have `Create` as an option. We had added the way to open Publisher in Create tab for convenience.

```
lambda: host_tools.show_publisher(
    tab="create"
)
```

Reply:
Will add the Creator button back. It confused me a bit when it disappeared. 
Also the big thank should go to @BigRoy as he's the one who have done 90% of all the work here :)

Reply:
- Nuke seems to have this Create for the new publisher
- Houdini implementation does seem to have the Create menu entry but it [actually still seems to trigger `show_creator` to the old creator](https://github.com/ynput/OpenPype/blob/develop/openpype/hosts/houdini/startup/MainMenuCommon.xml#L7-L15)? Likely a bug?
- Quickly glancing over the code for After Effects and Photoshop, it doesn't seem to have the Create option?
- The [TVPaint implementation doesn't seem to have the Create menu](https://github.com/ynput/OpenPype/blob/118cf08d91e9fd3a4a540a5bf5d118198965fad6/openpype/hosts/tvpaint/api/communication_server.py#L837-L866) entry either
I feel like we should at least UNIFY it across all hosts. 

Looking at the above it actually makes more sense to also clean them up from Nuke + Houdini?

Which has our preference? Maybe @mkolar @iLLiCiTiT want to make a call on that too? I recall this discussion has come up before but I can't seem to find it now.

Reply:
My vote goes to have a Create button. Makes it very clear where you will end up.
Else to create an object you'll have to press publish and in the new UI go to the tab Create.

The Create tab it maybe the wrong location to place non-publishable things but you could place creations like formatting-presets and lensflare-presets. Things that aren't really an asset as it's built by nodes but things that won't be published later on.

Reply:
Hmm maybe I'm seeing the issue in reverse -  but for me clicking "Publish" in Fusion now actually already defaults to the Create tab. :) So that should be changed to the publish tab then I suppose?

Anyway, if others have input please drop it here now - would love to sort this out soon. :)